sdk/keyvault/Microsoft.Azure.KeyVault.Core/src/Microsoft.Azure.KeyVault.Core.csproj

```diff
@@ -18,7 +18,7 @@ It is recommended that you move to the new package.</Description>
         Added recommendation to use new libraries listed in the package description.
       ]]>
     </PackageReleaseNotes>
-    <Version>3.0.5</Version>
+    <Version>3.1.0-beta.1</Version>
```

Comment:
Though I'd prefer not to touch these files, these track 1 packages are officially deprecated. We'd probably never release a new version even with a critical bug, but if we did it'd be a new patch release - not a minor release.

Reply:
@chidozieononiwu why isn't the old state valid? It had a matching version and the date was stamped in. What part of the validation was failing?

Reply:
This particular case is invalid because the entry has no content.

Reply:
While the chances we ever ship updates to the Key Vault track 1s is almost nil, this version should still not be a minor version bump.

Reply:
So should that be `3.0.6`?

Reply:
3.0.6-beta1, probably.

Reply:
I'd suggest we just fix the invalid entry and add some text there instead of changing the versions.

Reply:
@heaths  can you suggest what I should update the existing version with?

Reply:
I +1'd your [comment above](https://github.com/Azure/azure-sdk-for-net/pull/28174#discussion_r849870817). Just go ahead and set it to 3.0.6. We've 9.999% never release it again but if we do, at least it's following the same practices we do for track 2s.

Reply:
OK if we plan to increment patch that is fine but we should not add the beta.1